conf/messages

```diff
@@ -1,5 +1,9 @@
 # https://www.playframework.com/documentation/latest/ScalaI18N
 
+nationalArchives.emailForHtmlTemplates=tdr@nationalarchives.gov.uk
+nationalArchives.judgmentsEmailForHtmlTemplates=tdr@nationalarchives.gov.uk
+judiciary.emailForHtmlTemplates=Judgments@@judiciary.uk
+
```

Comment:
I think you don't need to use the `ForHtmlTemplates`.

nationalArchives.email=tdr@nationalarchives.gov.uk
nationalArchives.judgmentsEmail=tdr@nationalarchives.gov.uk
judiciary.email=Judgments@@judiciary.uk